.github/workflows/build-flannel-images.yml

```diff
@@ -0,0 +1,19 @@
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow.json
+name: Build Flannel images
+
+on:
+  workflow_dispatch:
```

Comment:
Should we add any other triggers?
What about on push to the default branch?

Am I right that workflow_dispatch means that it must be triggered manually?
If so, when will we trigger a manually build?

Reply:
Yes `workflow_dispatch` means triggered manually.

If we push on default branch we'd need another versioning strategy.
Right now the flannel version is used for the container image tag / version too and I think should avoid changing the contents of images we already pushed. If we push every time there are updates the default branch (even if we limit the job to changes under hostprocess/flannel) we would be updating the flannel images in between flannel releases.

Let's discuss triggers in a separate issue?

Reply:
Let's do so :)